renovate-presets/branch-merge.json

```
@@ -7,7 +7,7 @@
                 "digest"
             ],
             "automergeSchedule": [
-                "after 9am and before 2pm every weekday"
+                "* 9-14 * * 1-5"
```

question: I think it should be:
```suggestion
                "* 9-13 * * 1-5"
```

According to renovate:
```
after 10pm and before 5am every weekday	* 22-23,0-4 * * 1-5
```
Why?
Renovate does not support scheduled minutes or "at an exact time" granularity. Granularity must be at least one hour.

---

Yes, you're right